Gemfile

```diff
@@ -17,7 +17,7 @@ group :test do
   gem 'rspec'
   gem 'beaker', *location_for(ENV['BEAKER_VERSION'] || '~> 2.9.0')
   if ENV['GEM_SOURCE'] =~ /rubygems\.delivery\.puppetlabs\.net/
-    gem 'sqa-utils', '~> 0.11'
+    gem 'sqa-utils', '0.12.1'
```

Comment:
This is a merge-up commit.  None of our other branches are using `0.13`, and none of them are failing tests, so I don't think we should make that change as part of this PR.

That's not to say that we shouldn't do it soon/right away, but I'd like to keep this PR focused strictly on merging and addressing merge conflicts.